.github/workflows/build.yml

```diff
@@ -32,15 +32,21 @@ jobs:
 
     steps:
       - name: Checkout
-        uses: actions/checkout@v2
+        uses: actions/checkout@v4
 
       - name: Set up QEMU
-        uses: docker/setup-qemu-action@v1
+        uses: docker/setup-qemu-action@v3
 
-      - name: Set up Docker Buildx
-        uses: docker/setup-buildx-action@v1
+      - name: Set up Docker
+        uses: docker/setup-docker-action@v4
         with:
-          driver: docker
+          daemon-config: |
+            {
+              "debug": true,
+              "features": {
+                "containerd-snapshotter": true
```

Comment:
It's not possible to load non-native builds by default, we need to enable the containerd storage to support this.